--- scotty-00-02-21/tkined/generic/tkiInit.c~	Sun Sep 22 18:09:09 2002
+++ scotty-00-02-21/tkined/generic/tkiInit.c	Sun Sep 22 18:09:09 2002
@@ -73,14 +73,6 @@
 
     buffersize(1024);
 
-    if (Tcl_PkgRequire(interp, "Tcl", TCL_VERSION, 1) == NULL) {
-        return TCL_ERROR;
-    }
-
-    if (Tcl_PkgRequire(interp, "Tk", TK_VERSION, 1) == NULL) {
-        return TCL_ERROR;
-    }
-
     code = Tcl_PkgProvide(interp, "Tkined", TKI_VERSION);
     if (code != TCL_OK) {
         return TCL_ERROR;
--- scotty-00-02-21/tnm/generic/tnmInit.c~	Sun Sep 22 18:12:04 2002
+++ scotty-00-02-21/tnm/generic/tnmInit.c	Sun Sep 22 18:12:04 2002
@@ -434,10 +434,6 @@
     }
 #endif
     
-    if (Tcl_PkgRequire(interp, "Tcl", TCL_VERSION, 1) == NULL) {
-        return TCL_ERROR;
-    }
-
     if (Tcl_PkgProvide(interp, "Tnm", TNM_VERSION) != TCL_OK) {
         return TCL_ERROR;
     }
