--- scotty-00-02-21/unix/configure.in~	Wed Oct 18 02:19:26 2000
+++ scotty-00-02-21/unix/configure.in	Wed Oct 18 02:20:24 2000
@@ -79,9 +79,9 @@
 
 AC_MSG_CHECKING(for tcl version)
 AC_MSG_RESULT("$TCL_VERSION")
-if test "$TCL_VERSION" != "8.2" ; then
-    AC_MSG_ERROR("Tcl version 8.2 needed")
-fi
+#if test "$TCL_VERSION" -lt 8.2 ; then
+#    AC_MSG_ERROR("Tcl version 8.2 needed")
+#fi
 
 BUILD_TARGETS="$BUILD_TARGETS scotty"
 INSTALL_TARGETS="$INSTALL_TARGETS tnm-install"
@@ -127,9 +127,9 @@
 
     AC_MSG_CHECKING(for tk version)
     AC_MSG_RESULT("$TK_VERSION")
-    if test "$TK_VERSION" != "8.2" ; then
-        AC_MSG_ERROR("Tk version 8.2 needed")
-    fi
+#    if test "$TK_VERSION" != "8.2" ; then
+#        AC_MSG_ERROR("Tk version 8.2 needed")
+#    fi
 
     BUILD_TARGETS="$BUILD_TARGETS tkined"
     INSTALL_TARGETS="$INSTALL_TARGETS tki-install"
@@ -312,7 +312,7 @@
     exit(1);
   if (strcmp(SMI_LIBRARY_VERSION, smi_library_version))
     exit(2);
-  if (strcmp("2:1:0", smi_library_version))
+  if (strcmp("2:1:0", smi_library_version) > 0)
     exit(2);
   exit(0);
 }
