diff -ur scotty-00-02-21.orig/unix/Makefile.in scotty-00-02-21/unix/Makefile.in
--- scotty-00-02-21.orig/unix/Makefile.in	Wed Feb 16 17:08:00 2000
+++ scotty-00-02-21/unix/Makefile.in	Wed Oct 18 01:56:18 2000
@@ -28,22 +28,22 @@
 exec_prefix =	@exec_prefix@
 
 # Directory in which to install the program scotty:
-BIN_INSTALL_DIR =	$(exec_prefix)/bin
+BIN_INSTALL_DIR =	$(DESTDIR)$(exec_prefix)/bin
 
 # Directory in which to install dynamic loadable modules:
-LIB_INSTALL_DIR =       $(exec_prefix)/lib
+LIB_INSTALL_DIR =       $(DESTDIR)$(exec_prefix)/lib
 
 # Directory in which dynamic loadable modules are installed:
-LIB_RUNTIME_DIR =       $(exec_prefix)/lib
+LIB_RUNTIME_DIR =       $(DESTDIR)$(exec_prefix)/lib
 
 # Directory in which to install library files belonging to the extension.
-TNM_INSTALL_DIR =	$(prefix)/lib/tnm$(TNM_VERSION)
+TNM_INSTALL_DIR =	$(DESTDIR)$(prefix)/lib/tnm$(TNM_VERSION)
 
 # Directory in which to install the library of tkined scripts.
-TKI_INSTALL_DIR =	$(prefix)/lib/tkined$(TKI_VERSION)
+TKI_INSTALL_DIR =	$(DESTDIR)$(prefix)/lib/tkined$(TKI_VERSION)
 
 # Top-level directory for manual entries:
-MAN_INSTALL_DIR =	$(prefix)/man
+MAN_INSTALL_DIR =	$(DESTDIR)$(prefix)/share/man
 
 # Directory in which to install the manual entries:
 MAN1_INSTALL_DIR =	$(MAN_INSTALL_DIR)/man1
@@ -589,10 +589,6 @@
 		    $(INSTALL_DATA) $$i $(TNM_INSTALL_DIR)/mibs ; \
 		done ; \
 	    done
-	@TCLLIBPATH="$(TNM_INSTALL_DIR) $$TCLLIBPATH"; \
-	export TCLLIBPATH; \
-	echo 'foreach f $$tnm(mibs) {puts "Parsing $$f"; Tnm::mib load $$f}; exit' | \
-		./scotty
 
 tnm-install-library:
 	@for i in $(TNM_INSTALL_DIR)/library ; \
@@ -803,10 +799,6 @@
 	    echo "Installing $$i"; \
 	    $(INSTALL_DATA) $$i $(TKI_INSTALL_DIR)/apps; \
 	    done
-	@TCLLIBPATH="$(TNM_INSTALL_DIR) $$TCLLIBPATH"; \
-	export TCLLIBPATH; \
-	echo 'cd $(TKI_INSTALL_DIR)/apps; auto_mkindex . library.tcl; exit' | \
-		$(TCL_TCLSH)
 
 sinstall: nmicmpd nmtrapd
 	@for i in $(BIN_INSTALL_DIR) ; \
@@ -813,7 +813,7 @@
 	@for i in nmicmpd nmtrapd ; \
 	    do \
 	    echo "Installing $$i"; \
-	    $(INSTALL_PROGRAM) -o root -m 4755 $$i $(BIN_INSTALL_DIR)/$$i; \
+	    $(INSTALL_PROGRAM) $$i $(BIN_INSTALL_DIR)/$$i; \
 	    done
 
 uninstall: tnm-uninstall tnm-uninstall-man tki-uninstall tki-uninstall-man
